src/Common/Exceptions/ServiceException.php

```diff
@@ -86,7 +86,12 @@ protected static function parseErrorMessage(ResponseInterface $response)
         $serializer = new XMLSerializer();
         $errorMessage = '';
         try {
+            $internalErrors = libxml_use_internal_errors(true);
             $parsedArray = $serializer->unserialize($response->getBody());
+            if (libxml_get_errors()) {
+                throw new \Exception('Cannot parse XML');
```

Comment:
The message is in `Resources` now, and contains errors coming from `libxml_get_errors()`. The exception is caught few lines below and the message is discarded, but I agree it's better to have a meaningful message in the exception.